scripts/process/merge_centroids_wu.R

```diff
@@ -11,8 +12,9 @@ process.merge_centroids_wu <- function(viz) {
     left_join(wu_data_15_simple, by=c('GEOID'='FIPS')) %>%
     select(GEOID, STATE_ABBV, COUNTY, countypop:industrial) %>% 
     rowwise() %>% 
-    mutate(other = total - sum(thermoelectric, publicsupply, irrigation, industrial))
-  
+    mutate(other = total - sum(thermoelectric, publicsupply, irrigation, industrial)) %>%
+    mutate(total = as.integer(round(total)))
```

Comment:
I think we should make a style decision about how many digits to show that allows us to be consistent between total and other categories, and at any spatial resolutions we choose to show. If the raw data come in at 2 digits after the decimal, maybe we always show exactly 2 digits? or we could choose to show all categories as integers; that would be a little cleaner, though I imagine the WU team might be sad about giving up those hard-won final digits.

Reply:
Yeah, I was going to address all the rounding in another PR. For whatever reason...it's always trickier (for me?) than it seems. I figured though that integers in total were probably OK (since we know those are the biggest numbers.... and that prevents the numerous 10.0000000000003 )